CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v0.31.5
+
+*April 16th, 2019*
+
+This release fixes a regression from v0.31.4 where, in existing chains that
+were upgraded, `/validators` could return an empty validator set. This is true
+for almost all heights given the validator set remains the same.
+
+Special thanks to external contributors on this release:
+@brapse, @guagualvcha, @dongsam, @phucc
+
+### IMPROVEMENTS:
+
+- [libs/common] CMap: slight optimization in Keys() and Values() (@phucc)
```

Comment:
Nit:
```suggestion
- [libs/common] `CMap`: slight optimization in `Keys()` and `Values()` (@phucc)
```
(We do not consistently mark code / symbols as such but I think we should).